src/plugins/navigation/public/top_nav_menu/top_nav_menu.tsx

```diff
@@ -97,10 +110,12 @@ export function TopNavMenu(props: TopNavMenuProps): ReactElement | null {
   }
 
   function renderMenu(className: string): ReactElement | null {
-    if (!config || config.length === 0) return null;
+    if ((!config || config.length === 0) && (!showDataSourceMenu || !dataSourceMenuConfig))
```

Comment:
There is typo in this file with line#124 `// Validate presense of all required fields`, `presense` should be `presence`

Reply:
Yeah, fixed the comment